htdocs/lib2/logic/cachelist.class.php

```diff
@@ -20,7 +20,7 @@ class cachelist
     public $nCachelistId = 0;
     public $reCachelist;
 
-    public function __construct($nNewCachelistId = ID_NEW, $nUserId = 0)
+    public function __construct($nNewCacheListId = ID_NEW, $nUserId = 0)
```

Comment:
Sry, forget that. I just verified that the English term is "cache list", and that becomes "CacheList" in PHP speak.